avocado/utils/network/interfaces.py

```diff
@@ -149,7 +149,7 @@ def is_admin_link_up(self):
                  otherwise will return False.
         """
         try:
-            if self._get_interface_details().get('flags')[2] == 'UP':
+            if self._get_interface_details().get('flags')[-2] == 'UP':
```

Comment:
Actually I was thinking about how to improve this. Maybe  we could have something like:

```python
if ' UP' in  self._get_interface_details().get('flags'):
```

Reply:
@beraldoleal this also be fine, we can do this.